Documentation/Developer/InterlinkInventories.rst

```diff
@@ -0,0 +1,102 @@
+*   :ref:`TYPO3 Explained, stable version (for example 12.4) <t3coreapi_stable:start>`
+*   :ref:`TYPO3 Explained, old stable version (for example 11.5) <t3coreapi_oldstable:start>`
+
+The preferred version can be set in the guides.xml to `dev`, `stable`
+`oldstable` or a specific minor version, for example `8.7`.
+
+..  code-block:: xml
+    :caption: Documentation/guides.xml
+
+    <?xml version="1.0" encoding="UTF-8" ?>
+    <guides
+        xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+    >
+        <project title="Render guides"/>
+        <extension
+            class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+            typo3-core-preferred="stable"
+        />
+    </guides>
+
+or
+
+..  code-block:: xml
+    :caption: Documentation/guides.xml, excerpt
+
+    <extension
+        class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+        typo3-core-preferred="8.7"
+
+It is not necessary anymore to list each of the standard inventories in the
+guides.xml anymore. If desired you can override or redefine standard interlink
+inventories or define new ones:
+
+..  code-block:: xml
+    :caption: Documentation/guides.xml
+
+    <?xml version="1.0" encoding="UTF-8" ?>
+    <guides
+        xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+    >
+        <project title="Render guides"/>
+        <extension
+            class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+            typo3-core-preferred="stable"
+        />
+        <!-- explicitly link to version 8.7 of TYPO3 Explained -->
+        <inventory id="t3coreapi_v8" url="https://docs.typo3.org/m/typo3/reference-coreapi/8.7/en-us/"/>
+        <!-- ext_sys_note and ext_sys_note_stable should always link to main-->
+        <inventory id="ext_sys_note" url="https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/"/>
+        <inventory id="ext_sys_note_stable" url="https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/"/>
+    </guides>
+
+The following links:
+
+..  code-block:: rst
+
+    *   :ref:`TYPO3 Explained, always version 8.7 <t3coreapi_v8:start>`
+    *   :ref:`Sys note always goes to main <ext_sys_note:start>`
+    *   :ref:`Sys note always goes to main <ext_sys_note_stable:start>`
+
+*   :ref:`TYPO3 Explained, always version 8.7 <t3coreapi_v8:start>`
+*   :ref:`Sys note always goes to main <ext_sys_note:start>`
+*   :ref:`Sys note always goes to main <ext_sys_note_stable:start>`
+
+
+Adding a new TYPO3 version or manual
+====================================
+
+When a new TYPO3 Version is released or the long time support changes you
```

Comment:
```suggestion
When a new TYPO3 version is released or the long-time support changes you
```

I would avoid the "you", maybe ... "the long-time support changes the according TYPo3 versions can be adjusted directly in the theme in the enum" or something. This makes it clear it is nothing for the average user (although, this is about the render-guides, but to make it clear).